mlflow/server/js/src/model-registry/components/CompareModelVersionsView.js

```diff
@@ -36,18 +42,27 @@ export class CompareModelVersionsView extends Component {
     runDisplayNames: PropTypes.arrayOf(PropTypes.string).isRequired,
     modelName: PropTypes.string.isRequired,
     versionsToRuns: PropTypes.object.isRequired,
+    inputsListByName: PropTypes.arrayOf(Array).isRequired,
+    inputsListByIndex: PropTypes.arrayOf(Array).isRequired,
+    outputsListByName: PropTypes.arrayOf(Array).isRequired,
+    outputsListByIndex: PropTypes.arrayOf(Array).isRequired,
```

Comment:
```suggestion
    inputsListByName: PropTypes.arrayOf(PropTypes.array).isRequired,
    inputsListByIndex: PropTypes.arrayOf(PropTypes.array).isRequired,
    outputsListByName: PropTypes.arrayOf(PropTypes.array).isRequired,
    outputsListByIndex: PropTypes.arrayOf(PropTypes.array).isRequired,
```

We need to pass a propTypes object to `PropTypes.arrayOf` otherwise it validates nothing.

Reply:
Thanks for pointing that out!